### DISCUSSION TITLE
How to add second bold line of text?

### DISCUSSION THREAD

**Discussion by thecarpetjasp** [Windos/BurntToast#200] [Q&A]
So when you add text like so: 
                                                  "first line", 'second line'
The first line is in bold and the second line is not bold.
Is there a way to add a second line of bold so that both first and second line are bold?

**Comment by Windos** [comment]
Unfortunately, you can't. The first is automatically considered a heading of sorts and the rest are "additional". This is all handled by Windows (we're just telling it, "there's x lines of text" and Windows does its thing.)

You may be able to do something with "columns" (in this case, just one) as that has additional styles available for the test elements... but it won't be the same as just having two bold lines of text.

There's an example of these columns over in my [Talks repo](https://github.com/Windos/Talks/blob/main/2021/02%20-%20February/23%20-%20DuPSUG%20-%20Croque%20Monsieur/Demos/10_Columns.ps1)

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
